### PR TITLE
 BACKLOG-21276 : migrate vanity URL on file test

### PR DESCRIPTION
**Pull request by ksagnard**
### Summary by CodeRabbit

- New Feature: Added a new end-to-end test case "Add a first basic vanity URL on a file". This test case covers the functionality of adding a vanity URL to a file, publishing the changes, and verifying the response status and content type of the vanity URL. This will help ensure the robustness and reliability of the vanity URL feature in the application.
- Chore: Updated import statements in test files for better modularity and maintainability.

### CODE REVIEW

File: tests/cypress/e2e/edit/addVanityUrlonFileUi.cy.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { publishAndWaitJobEnding, editSite } from '@jahia/cypress'
+import { ContentEditorSEO } from '../../page-object/ContentEditorSEO'
```
[the remark this answers is not in the document]
Comment (ksagnard) [reply tests/cypress/e2e/edit/addVanityUrlonFileUi.cy.ts:2]: editSite is used at line 14, publishAndWaitJobEnding at line 34

File: tests/cypress/e2e/edit/addVanityUrlonFileUi.cy.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        publishAndWaitJobEnding(sitePath + filePath + fileName)
+
+        cy.request(Cypress.env('JAHIA_URL') + '/vanityFile1').then((response) => {
+            // expect(response.body).to.have.length(500)
+            const mime = response.headers['content-type']
+            expect(mime).to.contains('image/jpeg')
+            const length = response.headers['content-length']
+            expect(length).to.eq('83342')
+        })
+    })
```
[the remark this answers is not in the document]
Comment (ksagnard) [reply tests/cypress/e2e/edit/addVanityUrlonFileUi.cy.ts:43]: fixed

**Review by dgriffon — CHANGES_REQUESTED** [review]
I suppose this match an existing test scenario that has been migrated.
Is it possible though to add few steps to bring more values to the test:
- check first that before the vanity to be set the requested url returns a 404
- remove the vanity, then check that the file is still accessible with its full url, then check that it returns a 404 with a vanity 

if it is out of scope, ignore my comment

**Comment by ksagnard** [comment]
i created [BACKLOG-21642](https://jira.jahia.org/browse/BACKLOG-21642) to handle David's comment, it would be another task

**Review by dgriffon — APPROVED** [review]

**Pull request merged**